app-catalog/src/api/charts.tsx

```diff
@@ -1,23 +1,39 @@
 import { PAGE_OFFSET_COUNT_FOR_CHARTS } from '../components/charts/List';
 
-export function fetchChartsFromArtifact(
+export async function fetchChartsFromArtifact(
   search: string = '',
+  verified: boolean,
   category: { title: string; value: number },
   page: number,
   limit: number = PAGE_OFFSET_COUNT_FOR_CHARTS
 ) {
+  // note: we are currently defaulting to search by verified and official as default
+
   if (!category || category.value === 0) {
-    return fetch(
+    const response = await fetch(
       `https://artifacthub.io/api/v1/packages/search?kind=0&ts_query_web=${search}&sort=relevance&facets=true&limit=${limit}&offset=${
         (page - 1) * limit
-      }`
-    ).then(response => response.json());
+      }&verified_publisher=${verified}`
```

Comment:
There's a way to work links that's a bit easier to manage. You can use URL object like this 

js
```
const url = new URL('https://artifacthub.io/api/v1/packages/search');
url.searchParams.set('kind', '0');
url.searchParams.set('ts_query_web', search);
url.searchParams.set('sort', 'relevance');
url.searchParams.set('facets', 'true');
url.searchParams.set('limit', limit);
url.searchParams.set('offset', (page - 1) * limit);
url.searchParams.set('verified_publisher', verified);
```
and then you can use url as a string `fetch(url.toString())`